src/gocam/cli.py

```diff
@@ -476,5 +480,273 @@ def flatten_models(input_file, input_format, output_format, output_file, fields)
+        gocam translate-collection
+        
+        # Only NetworkX format with custom output directory
+        gocam translate-collection --formats networkx --output-networkx ./my_output
+        
+        # Test with limited models
+        gocam translate-collection --limit 5
+    """
+    def download_and_extract_tarball(url: str, extract_dir: str) -> str:
+        """Download and extract a gzipped tarball."""
+        logger.info(f"Downloading tarball from {url}")
+        
+        with tempfile.NamedTemporaryFile(suffix='.tgz', delete=False) as tmp_file:
+            try:
+                urlretrieve(url, tmp_file.name)
+                logger.info(f"Downloaded tarball to {tmp_file.name}")
+                
+                # Extract the tarball
+                logger.info(f"Extracting tarball to {extract_dir}")
+                with tarfile.open(tmp_file.name, 'r:gz') as tar:
+                    tar.extractall(path=extract_dir)
+                    
+                    # Find the extracted directory
+                    extracted_dirs = [d for d in os.listdir(extract_dir) 
+                                    if os.path.isdir(os.path.join(extract_dir, d))]
+                    if extracted_dirs:
+                        return os.path.join(extract_dir, extracted_dirs[0])
+                    else:
+                        return extract_dir
+                        
+            finally:
+                # Clean up the temporary file
+                os.unlink(tmp_file.name)
+
+    def find_json_models(directory: str) -> List[str]:
+        """Find all JSON model files in a directory."""
+        json_files = []
+        for root, dirs, files in os.walk(directory):
+            for file in files:
+                if file.endswith('.json'):
+                    json_files.append(os.path.join(root, file))
+        
+        logger.info(f"Found {len(json_files)} JSON model files")
+        return json_files
+
+    def load_model_from_file(file_path: str) -> Optional[Model]:
+        """Load a GO-CAM model from a Minerva JSON file."""
+        try:
+            with open(file_path, 'r') as f:
+                minerva_dict = json.load(f)
+                # Convert from Minerva format to GO-CAM model
+                model = MinervaWrapper.minerva_object_to_model(minerva_dict)
+                return model
+        except Exception as e:
+            logger.error(f"Failed to load model from {file_path}: {e}")
+            return None
+
+    def translate_to_networkx(model: Model, output_path: str, model_filename: str):
+        """Translate a model to NetworkX format and save as JSON."""
+        try:
+            # Initialize the translator
+            indexer = Indexer()
+            translator = ModelNetworkTranslator(indexer=indexer)
+            
+            # Translate the model
+            json_output = translator.translate_models_to_json([model], include_model_info=True, indent=2)
```

Comment:
If these files are going to be served to downstream clients, I would avoid using `indent` here. These are pretty big JSON files and all those spaces and newlines add up. I did a quick test on a few CX2 files I had hanging around and the indented version is 2 to 2.5 times larger than the unindented version.

Reply:
ok.  so far its a pretty small set of outputs and I hate trying to debug unindented files, but I can manage with a client-side formatter.